test/unit/org/apache/cassandra/io/sstable/indexsummary/IndexSummaryTest.java

```diff
@@ -81,7 +82,7 @@ public static void setup()
     public void testIndexSummaryKeySizes() throws IOException
     {
         // On Circle CI we normally don't have enough off-heap memory for this test so ignore it
-        Assume.assumeTrue(System.getenv("CIRCLECI") == null);
+        Assume.assumeTrue(CIRCLECI.getString() == null);
```

Comment:
Wondering why we just don't assume exact amount of available direct memory?

Reply:
i would just leave it as it is ... there might be different executors I guess with different memory amount etc?

Reply:
not a suggestion to change in this ticket, just wondering...